metabase/metabase
# fix y-axis split on native questions

Closes https://github.com/metabase/metabase/issues/41283

### Description

Fixes auto y-axis split on native queries.

### How to verify

Ensure the following query produces a chart with two axes
```sql
SELECT
  products.category AS "x",
  COUNT(*) AS "m1",
  AVG(orders.discount) AS "m2"
FROM
  orders
JOIN products ON orders.product_id = products.id
GROUP BY
  products.category
```

### Demo

Before
<img width="972" alt="Screenshot 2024-04-16 at 9 25 44 PM" src="https://github.com/metabase/metabase/assets/14301985/1e114c0b-bf95-4aa5-b380-8708c1ce523d">

After
<img width="964" alt="Screenshot 2024-04-16 at 9 25 19 PM" src="https://github.com/metabase/metabase/assets/14301985/4688b893-8d97-4f6d-b761-9a0972874a60">

### Checklist
